test/format/t.c

```diff
@@ -340,13 +340,15 @@ main(int argc, char *argv[])
         testutil_assert(ret == 0 || ret == WT_NOTFOUND);
         ret = timestamp_query("get=stable_timestamp", &g.stable_timestamp);
         testutil_assert(ret == 0 || ret == WT_NOTFOUND);
+        g.prepared_id = 1;
```

Comment:
can we find a place for this in function `config_run`.

Reply:
done